DXRModules/DeusEx/Classes/DXREvents.uc

```diff
@@ -1486,6 +1486,79 @@ function SetWatchFlags() {
     }
 }
 
+function bool FailIfCorpseNotHeld(class<#var(DeusExPrefix)Carcass> carcClass, string goal)
+{
+    local #var(PlayerPawn) p;
+
+    p = player();
+    if (POVCorpse(p.inHand) == None || ("DeusEx." $ carcClass.name) != POVCorpse(p.inHand).carcClassString) {
```

Comment:
I think this should work, and would also fix the HX compilation

```suggestion
    if (POVCorpse(p.inHand) == None || string(carcClass) != POVCorpse(p.inHand).carcClassString) {
```

Reply:
Yep, seems to work.